pkgs/applications/video/qstopmotion/default.nix

```
@@ -4,12 +4,12 @@
 
 stdenv.mkDerivation rec {
   pname = "qstopmotion";
-  version = "2.4.0";
+  version = "2.4.1";
   name = "${pname}-${version}";
 
   src = fetchurl {
-    url = "mirror://sourceforge/project/${pname}/Version_2_4_0/${name}-Source.tar.gz";
-    sha256 = "0pbyq6nrr9g3crlsng660768167s0fybvcpzbfc0w9kkhs2jwrr2";
+    url = "mirror://sourceforge/project/${pname}/Version_2_4_1/${name}-Source.tar.gz";
```

```suggestion
    url = "mirror://sourceforge/project/${pname}/Version_${builtins.replaceStrings ["."] ["_"] version}/${name}-Source.tar.gz";
```
One less thing to update each time; might also enable for automatic updates.